.github/workflows/build-test-ubuntu-arm64.yml

```diff
@@ -45,6 +46,7 @@ jobs:
             cxx-standard: 20
           - os: ubuntu22
             docker-image: meshlib/meshlib-ubuntu22-arm64:${{inputs.docker_image_tag}}
+            container-options: "--user root"
```

Comment:
why?

Reply:
need to install packages for build stubs (similar to pip-build workflow)